kubernetes/conf.yaml.example

```diff
@@ -11,6 +11,9 @@ instances:
   # reach the kubelet and cadvisor APIs unless the environment variable
   # KUBERNETES_KUBELET_HOST is found (if your node names can be resolved by pods, we recommend you set this variable to spec.nodeName through the downward API).
   #
+  # To enable Kube State metrics, please refer to kubernetes_states integration. For more information,
```

Comment:
Thanks @ojongerius ! While I agree for Kubernetes State Metrics, the integration-core integration here refers to `kubernetes_states` integration : https://github.com/DataDog/integrations-core/tree/bc390930cdc31ab06bf020e97c28436249017539/kubernetes_state